server/poll/transform.go

```diff
@@ -208,3 +224,41 @@ func (p *Poll) ToEndPollPost(bundle *utils.Bundle, authorName string, convert ID
 
 	return post, nil
 }
+
+// ToCard return the poll for rhs card
+func (p *Poll) ToCard(bundle *utils.Bundle, convert IDToNameConverter) string {
+	localizer := bundle.GetServerLocalizer()
+	s := fmt.Sprintf("# %s\n", p.Question)
+
+	creatorName, _ := convert(p.Creator)
+	s += fmt.Sprintf(bundle.LocalizeWithConfig(localizer, &i18n.LocalizeConfig{DefaultMessage: rhsCardPollCreatedBy})+" %s\n", creatorName)
+
+	const comma = ", "
+	for _, o := range p.AnswerOptions {
+		var voter string
+		if !p.Settings.Anonymous {
+			for i := 0; i < len(o.Voter); i++ {
+				displayName, err := convert(o.Voter[i])
+				if err != nil {
+					return ""
+				}
+				if i+1 == len(o.Voter) && len(o.Voter) > 1 {
+					voter += " " + bundle.LocalizeWithConfig(localizer, &i18n.LocalizeConfig{DefaultMessage: rhsCardPollVoterSeperator}) + " "
+				} else if i != 0 {
+					voter += comma
+				}
+				voter += displayName
+			}
+		}
+
+		s += "### " + localizer.MustLocalize(&i18n.LocalizeConfig{
```

Comment:
```suggestion
		s += "### " + bundle.LocalizeWithConfig(localizer, &i18n.LocalizeConfig{
```

Reply:
Done, i had to make some changes to unit tests.